src/routes/(app)/projects/_components/project-tile.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import type { Project } from '@/types/project';
+
+  let screenSizeThreshold = 768; // tailwindcss sets 48rem (768px) as the minimum with for mobile devices
+  let windowWidth: number;
+  export let project: Project;
+  export let isOdd: boolean;
+</script>
+
+<svelte:window bind:innerWidth={windowWidth} />
+
+<svelte:element
+  this={windowWidth < screenSizeThreshold ? 'a' : 'div'}
+  href={windowWidth < screenSizeThreshold ? 'https://lipsum.com' : undefined}
+  class="flex h-44 w-4/5 flex-row justify-center bg-[#521015]
+    {windowWidth < screenSizeThreshold
+    ? 'odd:self-start odd:rounded-r-xl even:self-end even:rounded-l-xl'
+    : null}
+    md:m-10 md:h-auto md:w-auto md:flex-none md:flex-col md:self-center md:bg-transparent lg:m-12"
+>
+  <div
+    class="md:basis-3/10 mx-5 mb-3 flex flex-row items-center gap-1 md:flex-col
```

Comment:
To fix uneven spacing in mobile (no visible changes in desktop I think)
```suggestion
    class="md:basis-3/10 w-full justify-around mx-5 mb-3 flex flex-row items-center gap-1 md:flex-col
```